--- ./Makefile.orig	2025-08-21 13:21:05.000000000 +0200
+++ ./Makefile	2025-09-02 13:13:00.686344312 +0200
@@ -12,7 +12,7 @@
 DEFINES = -DOPENSSL_SUPPRESS_DEPRECATED
 
 LDFLAGS = 
-LDLIBS = $(I2PD_PATH)/$(I2PD_LIB) -lboost_system$(BOOST_SUFFIX) -lboost_program_options$(BOOST_SUFFIX) -lssl -lcrypto -lz 
+LDLIBS = $(I2PD_PATH)/$(I2PD_LIB) -lboost_program_options$(BOOST_SUFFIX) -lssl -lcrypto -lz 
 
 
 ifeq ($(UNAME),Linux)
